ansible/roles/passenger/etc/apache2/.htpasswd

```diff
@@ -0,0 +1 @@
+cardreader@simple.org:$apr1$Z6nKZxqJ$W/u.2A3T7QiaoPIYKM3/90
```

Comment:
Can we put this in an ansible vault instead? Even having the hash in the git repo freaks me out (as this is open source!)

Reply:
Oh, yes! I thought I put it in ansible vault.
Thanks, I'll fix this.